### PR TITLE
Bursaries removal of content

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/sjaC5WTo/6675

### CODE REVIEW

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ inset_text:
         <br>
         <h2>Changes to eligibility and payment</h2>
 
-      <p>Trainees who train to teach in the 2024 to 2025 academic year are no longer eligible for the IRP. This applies to all trainees who started courses from 1 July 2024 onwards. Trainees in languages or physics are still eligible to <a href="/funding-and-support/scholarships-and-bursaries">get a bursary or apply for a scholarship</a> worth up to $scholarships_nonuk_max$.</p>
+      <p>Trainees who train to teach in the 2024 to 2025 academic year are no longer eligible for the IRP. This applies to all trainees who started courses from 1 July 2024 onwards. Trainees in certain languages may still be eligible to <a href="/funding-and-support/scholarships-and-bursaries">get a bursary or apply for a scholarship</a>.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:27]:
```suggestion
      <p>Trainees who train to teach in the 2024 to 2025 academic year are no longer eligible for the IRP. This applies to all trainees who started courses from 1 July 2024 onwards. Trainees in languages or physics may still be eligible to <a href="/funding-and-support/scholarships-and-bursaries">get a bursary or apply for a scholarship</a>.</p>
```

File: app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb [hunk 2]
```diff
@@ -21,15 +21,9 @@
     </ul>
     <%= render Content::ExpanderComponent.new(
     title: "bursaries and scholarships",
-    text: "If you train to teach languages or physics, you may be eligible for a bursary or scholarship. You're unlikely to be eligible for bursaries or scholarships in other subjects.",
+    text: "If you train to teach certain languages, you may be eligible for a bursary or scholarship. You're unlikely to be eligible for bursaries or scholarships in other subjects.",
```
Comment (emmaachesongray) [inline app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb:24]:
```suggestion
    text: "If you train to teach languages or physics, you may be eligible for a bursary or scholarship. You're unlikely to be eligible for bursaries or scholarships in other subjects.",
```

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md [hunk 3]
```diff
@@ -112,19 +112,19 @@ You will also need a [visa or immigration status](/non-uk-teachers/visas-for-non
 
 [Find teacher training courses](https://find-teacher-training-courses.service.gov.uk/).
 
-## Bursaries and scholarships for non-UK trainee languages and physics teachers
+## Bursaries and scholarships for non-UK languages and physics teachers
```
Comment (emmaachesongray) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md:115]:
```suggestion
## Bursaries and scholarships for non-UK trainee languages and physics teachers
```

Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md:115]:
```suggestion
## Bursaries and scholarships for non-UK trainee languages and physics teachers
```
This needs to go back into the h2

File: app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb (hunk 2 above)
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb:24]:
```suggestion
    text: "If you train to teach languages or physics, you may be eligible for a bursary or scholarship. You're unlikely to be eligible for bursaries or scholarships in other subjects.",
```

Comment (Sarah-DfE) [inline app/views/content/funding-and-support/scholarships-and-bursaries/_funding-widget.html.erb:24]:
```suggestion
    text: "If you train to teach certain subjects, you may be eligible for a bursary or scholarship. You're unlikely to be eligible for bursaries or scholarships in other subjects.",
```

File: app/views/content/landing/how-to-fund-your-teacher-training.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "How to fund your teacher training"
-description: Find out how you could fund your teacher training with bursaries and scholarships available up to $scholarships_generic_maxshortened$, depending on the subject you're training to teach.
+description: Find out how you could fund your teacher training with bursaries and scholarships, depending on the subject you're training to teach.
```
Comment (Sarah-DfE) [inline app/views/content/landing/how-to-fund-your-teacher-training.md:3]:
```suggestion
description: Find out how you could fund your teacher training with a bursary or scholarship, depending on the subject you're training to teach.
```

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md (hunk 1 above)
Comment (Sarah-DfE) [inline app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:27]:
```suggestion
      <p>Trainees who train to teach in the 2024 to 2025 academic year are no longer eligible for the IRP. This applies to all trainees who started courses from 1 July 2024 onwards. Trainees in certain subjects may still be eligible to <a href="/funding-and-support/scholarships-and-bursaries">get a bursary or apply for a scholarship</a>.</p>
```

File: app/views/content/steps-to-become-a-teacher/_step_2_understand_funding.html.erb [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 <p>You can apply for tuition fee and maintenance loans, even if you already have a student loan.</p>
 
-<p>If you're interested in teaching certain subjects, you might be able to get up to <%= v :scholarships_generic_maxshortened %> tax-free to support you while you're training. This money does not have to be paid back.</p>
+<p>If you're interested in teaching certain subjects, you might be able to get tax-free bursaries or scholarships. This money does not have to be paid back.</p>
```
Comment (Sarah-DfE) [inline app/views/content/steps-to-become-a-teacher/_step_2_understand_funding.html.erb:5]:
```suggestion
<p>If you're interested in teaching certain subjects, you might be able to get a tax-free bursary or scholarship. This money does not have to be paid back.</p>
```

**Review by Sarah-DfE — COMMENTED** [review]
Added some comments @emmaachesongray

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md (hunk 1 above)
Comment (gemmadallmandfe) [reply app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:27]: this isn't right - we should leave as in prod - `languages or physics`

**Review by spencerldixon — APPROVED** [review]
LGTM 👍

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me from a front end perspective

**Pull request merged**